interfaces/python/CMakeLists.txt.bak

```diff
@@ -0,0 +1,268 @@
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
This file looks like it was accidentally added?

Reply:
I will remove it.